cmrtlib/agnostic/share/cm_printf_host.cpp

```diff
@@ -626,10 +626,16 @@ bool PFParser::outputToken(const char *tkn, PCM_PRINT_HEADER header)
             switch (header->dataType)
             {
                 case CM_PRINT_DATA_TYPE_INT:
-                    m_args[m_numMultArg - m_argsExpected] = *((int*            )&(header->scalar64));
+                    {
+                        int* scalar_int = reinterpret_cast<int*>(&header->scalar64);
+                        m_args[m_numMultArg - m_argsExpected] = *scalar_int;
+                    }
                     break;
                 case CM_PRINT_DATA_TYPE_UINT:
-                    m_args[m_numMultArg - m_argsExpected] = *((unsigned int*   )&(header->scalar64));
+		    {
```

Comment:
seems tabs are used here. Can you replace them with 4 spaces? Please also keep the alignment. Thanks a lot

Reply:
My bad,  editor introduce a lot tabs that I don't aware. Fix soon.